validator_test.go

```diff
@@ -8675,11 +8675,105 @@ func TestEmail(t *testing.T) {
 	errs = validate.Var(s, "email")
 	Equal(t, errs, nil)
 
+	s = `"test test"@email.com` // U+00A0 No-Break Space between quotes
```

Comment:
I recommend replacing those test strings with code point values. This approach would make the string's exact contents immediately clear and unambiguous.
```suggestion
	s = "\"test\u00A0test\"@email.com"
```
https://go.dev/play/p/YBWM1EUvXoR